### ISSUE TITLE
Running ruff with lintquarto, it is not detecting settings in `pyproject.toml`

### ISSUE THREAD

**Issue by amyheather** [lintquarto/lintquarto#121]
When running:

```
uv run lintquarto -l ruff -p docs
```

It doesn't appear to be detecting these settings:

```
[tool.ruff]
exclude = [
  "tests/examples/*",
  "docs/pages/tools/examples/*",
]
line-length=79
format.quote-style="double"
lint.select = ["ALL"]
lint.ignore = [
  "T201",  # print found
  "T203",  # pprint found
  "S603",  # subprocess call: check for execution of untrusted input
  "COM812",  # Removed due to incompatability with ruff format
]
lint.pydocstyle.convention = "numpy"
lint.per-file-ignores."tests/*" = [
    "INP001",  # File {filename} is part of an implicit namespace package. Add an __init__.py.
    "S101",  # Use of `assert` detected
    "ANN",  # Missing type annotations
    "PLR2004",  # Magic value used in comparison
]
lint.per-file-ignores."docs/*" = [
    "INP001"
]
```

**Comment by sneakers-the-rat** [comment]
this appears to be fixed now? my changes to the pyproject.toml are being reflected in the linter checks

**Comment by amyheather** [comment]
Agreed - will close.
